### PR TITLE
Add compress topologies

### PR DESCRIPTION
**Pull request by dbaluta**
I was keeping this in our imx stable branch for too long. I think they are upstream quality although we need to rework compress parameters passing. We now use compress like parameters piggybacked on PCM params.

### CODE REVIEW

File: tools/topology/topology1/development/sof-imx8mp-compr-aac-wm8960.m4 [hunk 1]
```diff
@@ -0,0 +1,98 @@
+#
+# Topology with codec_adapter processing component for i.MX8QM/i.MX8QXP
```
Comment (iuliana-prodan) [inline tools/topology/topology1/development/sof-imx8mp-compr-aac-wm8960.m4:2]: Can you, please, update the target name for both 8MP topologies.

Comment (dbaluta) [reply tools/topology/topology1/development/sof-imx8mp-compr-aac-wm8960.m4:2]: sure.

Comment (dbaluta) [reply tools/topology/topology1/development/sof-imx8mp-compr-aac-wm8960.m4]: should be fine now.

Comment (iuliana-prodan) [reply tools/topology/topology1/development/sof-imx8mp-compr-aac-wm8960.m4]: It is! Thanks!

**Review by iuliana-prodan — APPROVED** [review]

**Review by paulstelian97 — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**